test/sql/storage/compact_block_size/default_block_size.test

```diff
@@ -9,7 +9,7 @@ require vector_size 2048
 statement error
 ATTACH 'data/storage/block_size_16kb.db' (TYPE DUCKDB, READONLY)
 ----
-Cannot read database file
+cannot initialize the same database with a different block size
```

Comment:
This should work - but maybe now it should throw a not implemented exception (but in the future this should work)